src/Node.lua

```diff
@@ -0,0 +1,8 @@
+local node = {}
```

Comment:
Because this is a class the variable name `Node` should be used (see http://lua-users.org/wiki/LuaStyleGuide#Naming ).